packages/frameworks-solid-start/package.json

```diff
@@ -15,39 +17,47 @@
     },
     "./client": {
       "types": "./client.d.ts",
+      "solid": "./client.js",
       "import": "./client.js"
     },
+    "./types": "./types.d.ts",
+    "./utils": {
+      "types": "./utils.d.ts",
+      "import": "./utils.js"
+    },
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "tsup --config ./tsup.config.js && node scripts/postbuild",
+    "build": "tsc -p . && rollup -c && node scripts/postbuild",
     "patch": "npm version patch --no-git-tag-version",
-    "clean": "rm -rf client.* index.*"
+    "clean": "rm -rf dist client.* index.* types.* utils.*"
   },
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
     "@auth/core": "workspace:*",
+    "@rollup/plugin-babel": "^6.0.3",
+    "@rollup/plugin-node-resolve": "^15.0.1",
     "@solidjs/meta": "^0.28.0",
     "@types/node": "^18.7.14",
-    "next-auth": "workspace:*",
+    "rollup": "^3.9.1",
     "solid-js": "^1.5.7",
-    "solid-start": "^0.2.14",
-    "tsup": "^6.5.0",
-    "typescript": "^4.8.2"
+    "solid-start": "^0.2.15",
+    "typescript": "^4.8.2",
+    "vite": "^4.3.1"
   },
   "peerDependencies": {
-    "@auth/core": "~0.2.2 || ^0.2.2",
+    "@auth/core": "0.5.1",
```

Comment:
[The Svelte package uses](https://github.com/nextauthjs/next-auth/blob/main/packages/frameworks-sveltekit/package.json#L47-L49)

```ts
  "dependencies": {
    "@auth/core": "workspace:*"
  },
```

instead of being a peerDep with a hardcoded version. Should we follow suit?

Reply:
The sveltekit version is wrong, ad the user also needs to install @auth/core on his project in order to get access for providers, @balazsorban44 said that in the future we will have an @auth/providers package

Reply:
https://github.com/OrJDev/next-auth/commit/f2b7afa65385b2c82365c22c805007d37393d14c

no longer hardcoding the version tho

Reply:
How about `"@auth/core": "*"`? Not sure why the `~ || ^` aren't working, but I currently get this in my own project:

![image](https://user-images.githubusercontent.com/109672176/233725044-14106cdf-d6d2-48db-ab33-f5221c1ad743.png)